openshift/4.0/scripts/oc-sonar-dotnet.sh

```diff
@@ -113,5 +112,4 @@ display_helper "${CMD_SONAR_BEGIN}" "${CMD_BUILD}" "dotnet test" "${CMD_SONAR_EN
 #   - copy results to common folder "../TestResults"
 #
 # The final `coverage.opencover.xml` is the one we want
-# -----------------------------------------------------------------------------------------
-exec $SHELL;
```

Comment:
@sookeke Is it alright if I remove this? this was causing the build to fail with:
**This account is currently not available.**